rs/types/management_canister_types/src/lib.rs

```diff
@@ -2586,6 +2586,13 @@ pub enum QueryMethod {
     FetchCanisterLogs,
 }
 
+impl QueryMethod {
+    /// Returns `true` if the query is directed to the management canister (subnet query).
+    pub fn is_management_canister_query(&self) -> bool {
```

Comment:
Is this necessary? `QueryMethod` is a management canister type for describing query methods for the management canister by definition, no? It's not used for any other queries, nor it's expected to, right?